docs/core/tools/dotnet-new.md

```diff
@@ -20,7 +20,9 @@ dotnet new <TEMPLATE> [--dry-run] [--force] [-i|--install {PATH|NUGET_ID}]
     [--nuget-source <SOURCE>] [-o|--output <OUTPUT_DIRECTORY>]
     [-u|--uninstall] [--update-apply] [--update-check] [Template options]
 
-dotnet new <TEMPLATE> [-l|--list] [--type <TYPE>]
+dotnet new <TEMPLATE> [-l|--list] [--type <TYPE>] [-lang|--language {"C#"|"F#"|VB}] [--tag <TAG>]
+
```

Comment:
@KathleenDollard could you please confirm if you want new options to appear in synopsis?

Reply:
We generally include all available options in the synopsis, and note in the option details entries any variations on which SDK versions they apply to.